### ISSUE TITLE
Why does v6 of this app set session[:shopify_domain] as well and not just session[:shopify]?

### ISSUE THREAD

**Issue by pjc** [Shopify/shopify_app#143]
I am upgrading my app from V5 to V6 of this app mainly to get out of the bootstrap dependency so I can use bootstrap 3.

In the [new SessionsController](https://github.com/Shopify/shopify_app/blob/dea2b944323ccfa1d84a92c932d1007f92e314cd/lib/shopify_app/sessions_controller.rb) we now set both session[:shopify_domain] and session[:shopify], while V5 apps only set session[:shopify].

Just to confirm: I don't see this session being used anywhere else in this repo except for tests, so I assume it's just optional?

**Comment by kevinhughes27** [comment]
Yup its totally optional - we are setting it since its pretty common for error handlers (like Errbit and Airbrake) to dump the session in with the exception. This way you can see the shop name as well as the ID.

tl;dr extra visibility of what shop is the current shop
